client/src/app/site/services/global-search/global-search.service.ts

```diff
@@ -48,6 +48,18 @@ export class GlobalSearchService {
         };
     }
 
+    public getTitle(collection: string, content: any) {
+        if (collection === `user`) {
+            const firstName = content.first_name?.trim() || ``;
+            const lastName = content.last_name?.trim() || ``;
+            const userName = content.username?.trim() || ``;
+            const name = firstName || lastName ? `${firstName} ${lastName}` : userName;
+            return `${content.title?.trim() || ``} ${name?.trim()}`.trim() || ``;
+        }
+
+        return content.title || content.name;
+    }
+
```

Comment:
Why did you move this code?

Reply:
We have the convention that public methods should be defined above private or protected ones. This is just cleanup.